src/CheckVariables.ts

```diff
@@ -1011,6 +1011,20 @@ export const checkVariablesOnLoad = (data: PlayerSave) => {
     }
   }
 
+  if (data.blueberryLoadouts === undefined) {
```

Comment:
exactly

Reply:
This is a deeper problem; if a user loads the game and their savefile does not have `blueberryLoadouts` defined in their save, it will not be set even if `Synergism.ts` player file has it set, so it will never be defined unless it is explicitly set in `CheckVariables.ts`.

Reply:
EDIT: This appears to be false, unlike what I conjected. Thanks for pointing it out!